### DISCUSSION TITLE
Different background shade for today's column

### DISCUSSION THREAD

**Discussion by t214c** [iSoron/uhabits#1239] [Feature Requests]
A cosmetic improvement:

![4](https://user-images.githubusercontent.com/41517489/146177054-49e199c0-e460-46a8-90f5-2bdb4d060092.png)

**Comment by bobfromtahoe** [comment]


**Reply by bobfromtahoe** [reply]
I like this suggestion. It would help users ensure they are updating today's data in the correct column

**Comment by 2011413125** [comment]
Or just an option to only show today's column.

**Comment by iSoron** [comment]
Thanks for the suggestion, @t214c. I think it looks nice. Do you also have a mockup for the light theme?

**Reply by t214c** [reply]
@iSoron Sorry for the late reply. How about this?

![](https://user-images.githubusercontent.com/41517489/189032830-3aae6207-a707-48a9-b631-4dbd13af98eb.png)

**Comment by herrbrixter** [comment]
Related to #851 

I'd like to have both ideas. Weekends and Today should both have different colors.
